packages/infra/lib/ihe-stack/ihe-gw-construct.ts

```diff
@@ -404,43 +407,3 @@ export default class IHEGatewayConstruct extends Construct {
   //   });
   // }
 }
-
-function addMetricsToTargetGroup(
```

Comment:
Moved to a [shared file](https://github.com/metriport/metriport/pull/2006/files#diff-f49e462dd18f0da9a3e674e82494418c65b8ae0774a99c0720c8f2ea6dfc1b94R8).